packages/app/src/embed/components/Actions/index.js

```diff
@@ -46,9 +46,14 @@ export function GlobalActions({
   );
 }
 
-export function NavigationActions({ refresh, openInNewWindow, isDragging }) {
+export function NavigationActions({
```

Comment:
Maybe make this a `FunctionComponent` with implicit return? 🤔

Reply:
What's the benefit?

Reply:
It's a React component, right? 🤔

Since we're converting every component to TypeScript and typing it as `FunctionComponent`, I think we should follow the "boyscout rule" and make this a `FunctionComponent`.

Reply:
sorry, I'm lost

2 questions:

1. this isn't a ts file yet, so can't type it just yet
2. does `FunctionComponent` require an explicit return?

Reply:
1. Indeed, I think you should make this a TS file
2. No it doesn’t enforce it, but it will make it have 1 less indentation

Reply:
1. I'm still learning typescript so not yet, but soon?
2. I'll skip the aesthetic value if that's alright